components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/DataServiceResource.java

```diff
@@ -163,4 +171,33 @@ private ServiceMetaData getServiceMetaData(DataService dataService) throws Excep
             return null;
         }
     }
-}
\ No newline at end of file
+
+    private List<ResourceInfo> getResources(DataService dataService) {
+
+        Set<Resource.ResourceID> resourceIDS = dataService.getResourceIds();
+        List<ResourceInfo> resourceList = new ArrayList<>();
+        for (Resource.ResourceID id : resourceIDS) {
+            ResourceInfo resourceInfo = new ResourceInfo();
+            Resource resource = dataService.getResource(id);
+            resourceInfo.setResourcePath(id.getPath());
+            resourceInfo.setResourceMethod(id.getMethod());
+            resourceInfo.setResourceQuery(resource.getCallQuery().getQuery().getQueryId());
```

Comment:
Do we need query info in the dashboard view? It is kind of an internal information. Isn't it?

Reply:
We need to show which query is related to the resource right? Please refer below.
<img width="1358" alt="Screen Shot 2020-06-25 at 9 46 47 AM" src="https://user-images.githubusercontent.com/22365810/85652852-77a1de00-b6c9-11ea-95de-08dacf84623b.png">

Reply:
It is an internal detail. We do not expose that from WSDL, etc. Instead we should expose query params.

Reply:
Modified as per the discussion. Added query params as well